### ISSUE TITLE
(shall we) publish everware at The Journal of Open Source Software

### ISSUE THREAD

**Issue by anaderi** [everware/everware#137]
http://joss.theoj.org/# ?

**Comment by amitkumarj441** [comment]
Yes! @anaderi . This is a nice idea! @betatim Did you initiated the request to JOSS for publishing [Everware](https://github.com/everware)!

**Comment by anaderi** [comment]
🆙

**Comment by betatim** [comment]
Go ahead if you want to submit it

**Comment by amitkumarj441** [comment]
@betatim But how can we submit it to JOSS! I don't know about the process, @anaderi I need your help?
